openff/toolkit/utils/serialization.py

```diff
@@ -318,7 +318,7 @@ def from_yaml(cls, serialized):
         yaml.SafeDumper.add_representer(
             OrderedDict,
             lambda dumper, value: self._represent_odict(
-                dumper, u"tag:yaml.org,2002:map", value
+                dumper, "tag:yaml.org,2002:map", value
```

Comment:
(not blocking) this diff looks scary but it just seems to be cosmetic https://stackoverflow.com/questions/2464959/whats-the-u-prefix-in-a-python-string

Reply:
I was a little spooked by this as well